app/services/add_receipt_to_c2_proposal_attributes.rb

```diff
@@ -0,0 +1,21 @@
+class AddReceiptToC2ProposalAttributes
```

Comment:
I think `perform`does make it conform with other service classes....but I think in the end this is just returning a hash. So really, a nice interface might be a model in `app/models` with a `to_h` method that returns the attributes (the ones currently returned by `perform`)